src/main/java/pl/pjwstk/woloappapi/controller/EventController.java

```diff
@@ -24,14 +23,25 @@ public class EventController {
     private final EventService eventService;
     private final EventMapper eventMapper;
 
-    @GetMapping()
-    public ResponseEntity<List<EventResponseDto>> getEvents() {
+    @GetMapping("")
+    public ResponseEntity<List<EventResponseDto>> getEvents(
+            @RequestParam(value = "language") String language) {
         List<Event> events = eventService.getAllEvents();
         List<EventResponseDto> eventDtos =
-                events.stream().map(eventMapper::toEventResponseDto).collect(Collectors.toList());
+                events.stream().map(e -> eventMapper.toEventResponseDto(e, languageToEnum(language))).collect(Collectors.toList());
         return new ResponseEntity<>(eventDtos, HttpStatus.OK);
     }
 
+    private Language languageToEnum(String language) {
+        return switch (language) {
+            case "pl" -> Language.PL;
+            case "en" -> Language.EN;
+            case "ua" -> Language.UA;
+            case "ru" -> Language.RU;
+            default -> null;

```

Comment:
co się stanie, jeśli jest null?

Nie powinien to być rzucony wyjątek? Nie oczekujemy innej wartości niż jeden z 4 języków. Każda inna opcja to błąd. Błąd to wyjątek, nie null.

Reply:
może tak?
https://www.baeldung.com/java-string-to-enum